tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -653,3 +661,30 @@ def process(self, raw, identifiers, frame_meta):
                 identifier, j, scores, x_mins, y_mins, x_maxs, y_maxs, meta
             ))
         return result
+
+
+class YoloxAdapter(YolorAdapter):
+    __provider__ = 'yolox'
+
+    @staticmethod
+    def boxes_postprocess(boxes, img_size):
+        grids = []
+        expanded_strides = []
+
+        strides = [8, 16, 32]
+        hsizes = [img_size[0] // stride for stride in strides]
+        wsizes = [img_size[1] // stride for stride in strides]
+
+        for hsize, wsize, stride in zip(hsizes, wsizes, strides):
+            xv, yv = np.meshgrid(np.arange(wsize), np.arange(hsize))
+            grid = np.stack((xv, yv), 2).reshape(1, -1, 2)
+            grids.append(grid)
+            shape = grid.shape[:2]
+            expanded_strides.append(np.full((*shape, 1), stride))
```

Comment:
as I can see, these calculation does not depend from model output. Maybe we can use some cache and recalculate grid only if image size is changed from previous step?

Reply:
Done